src/strategies/properties/correspondence.sml

```diff
@@ -196,3 +197,24 @@ fun allCorrespondenceMatches corrs qProps rProps =
     in
         newIdentities @ baseCorrs
     end;
+
+
+
+structure F = Correspondence.F;
+exception skipProp;
```

Comment:
Please make these "local", to avoid weird structures and exceptions leaking out. I.e., 
```
local
...
in
...
end;
```